tests/ext/sandbox/install_hook/hook_not_invoked_if_tracer_disabled.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Ensure hooks are not invoked after the tracer has been disabled
+--FILE--
+<?php
+
+function foo() {}
+DDTrace\install_hook("foo", function($h) {
+    print "Invoked\n";
+    ini_set("datadog.trace.enabled", 0);
+    $h->span(); // must not crash
```

Comment:
If I understand correctly, on the first call this is a dummy span, wouldn't it also make sense to test that the span isn't pushed?

Reply:
This is not testable as, when the tracer is disabled, the accessors are shortcircuited - pushing a span would crash, given it'd push onto destroyed structures anyway.